packages/doc/content/components/getting-started/default-web.mdx

```diff
@@ -0,0 +1,28 @@
+### Adding Gympass font in your project
+
+```javascript type=highlight
+import React from 'react';
+import GoogleFontLoader from 'react-google-font-loader';
+import { withTheme } from 'styled-components';
+
+const FontLoader = ({
+  theme: {
+    yoga: {
+      baseFont: { family, weight },
+    },
+  },
+}) => (
+  <>
+    {typeof window !== 'undefined' ? (
+      <GoogleFontLoader fonts={[{ font: family, weights: weight }]} />
+    ) : (
+      <link
+        rel="stylesheet"
+        href={`https://fonts.googleapis.com/css?family=${family}:${weight.join()}`}
+      />
+    )}
+  </>
+);
+
+export default withTheme(FontLoader);
+```
```

Comment:
Hi! @caioalexandrebr Thanks for your contribution!

In this case, The `FontLoader` component must be imported from `@gympass/yoga`. [Yoga already exports a component](https://github.com/Gympass/yoga/blob/master/packages/yoga/src/index.js#L21) to do this job, there's no need to add a new external library, like "react-google-font-loader".

So, In my opinion, I think that the example must be like the `ThemeProvider` component example.

Reply:
Hi @allyssonsantos !! :satisfied:

Very good your comment, I made the changes, now I think it is meeting expectations, what do you think?